### DISCUSSION TITLE
"Show people"

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#554] [General]
The "Show people" option is marked with asterisk meaning it requires restart in EP settings, but when the same option is toggled from the taskbar context menu, the "People" button appears immediately.

**Comment by valinet** [comment]
Shocker, like Earth is a sphere.

If you know how to make it toggle immediately, do not hesitate with a contribution.

**Comment by Anixx** [comment]
I have no idea, and thought that you can better see with your tools, what command is invoked from the context menu. I do not use this function anyway.

**Reply by valinet** [reply]
I don't know and I don't care that much either, I don't know who in this world uses that anyway. It's there just for the sake of being there. If I knew how to do it, I would have done it proper from the get go.

**Reply by valinet** [reply]
And I don't have any magic or special and secret "tools", it's patience (when it is), luck (when it is), a decompiler and lots of time figuring stuff out. Y'know, things accessible to the general population just fine, or so I believe.

**Reply by valinet** [reply]
Okay, fixed in 40.10. It does not require a restart anymore and actually works now, it got broken sometime in the past releases, but I guess no one noticed since no one actually uses this anyway.

**Comment by l0oo** [comment]
I guess literally no one is using this feature indeed but maybe Anixx just click it and noticed that "People" button appears, so reported. :)
IMHO not worth to lose time to fix such unimportant things (and maybe not worth to report them too). ;)

**Reply by valinet** [reply]
Yeah, maybe. I was a bit harsh initially, I admit that, like, I could have ignored; I too felt like such a thing is not worth reporting. Idk, I am a perfectionist, I understand the "sake of completeness", but on the other hand, nothing's ever complete and polishing each and every bit means working on this non-stop... I am trying to diversify my activities lately, so to speak. Emphasis on trying, as in not really succeeding.

**Reply by Anixx** [reply]
Yes, I also reported for the sake of perfectness. It would be OK if the option was removed altogether, especially given it exists in the taskbar context menu. But if it is implemented, it is presumed to work.

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio
